runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7609,7 +7609,9 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
              (TR::Options::getCmdLineOptions()->getOption(TR_EnableJITServerHeuristics) ||
              _compInfo.getPersistentInfo()->isLocalSyncCompiles()) &&
              !TR::Options::getCmdLineOptions()->getOption(TR_DisableUpgradingColdCompilations) &&
-             TR::Options::getCmdLineOptions()->allowRecompilation())
+             TR::Options::getCmdLineOptions()->allowRecompilation() &&
+             !entry->isDLTCompile() &&
```

Comment:
Maybe we should use `details->isMethodInProgress()` instead of `entry->isDLTCompile()` just for consistency with the rest of filters. Otherwise, it looks good.